rando.py

```diff
@@ -89,8 +91,34 @@ def cointoss(self, mess, args):
 
     @botcmd(admin_only=False)
     def diceroll(self, mess, args):
-        """Returns a dice number (1 through 6)"""
-        return random.randint(1, 6)
+        """Returns a 6-sided dice roll. Optional argument like '3w6 1w12'."""
+        if not args:
+            return random.randint(1, 6)
+        total = 0
+        rolls = list()
+        for d in re.finditer(_DICEROLLS, args):
+            if not d.group(1):
+                count = 1
+            else:
+                count = int(d.group(1))
+            sides = int(d.group(2))
+            if count > 100:
+                return "Sry, I don't have that many w%d in my bag" % sides
```

Comment:
Minor nitpick, but I'd prefer "Sry" -> "Sorry". The abbreviation seems unnecessary, and might confuse non-native speakers...